doc/source/dev/contributor/quickerstart_conda.rst

```diff
@@ -0,0 +1,25 @@
+.. _quickerstart-conda:
+
+==============================================================
+Development environment quickerstart guide (Linux and Mac)
+==============================================================
+
+With `Anaconda <https://www.anaconda.com/products/individual>`_ (and, on macOS,
+the `Xcode Command Line Tools <https://webkit.org/build-tools/>`_) installed,
+execute the following commands at the terminal from the base directory of
+your `SciPy <https://github.com/scipy/scipy>`_ clone::
+
+    # Best practice, use an environment rather than install in the base env
+    conda create -n my-env
+    conda activate my-env
+
+    # Prefer conda-forge (because it has the compilers we need)
+    conda config --env --add channels conda-forge
+
+    # Now install all packages that are needed in one `conda install` command
+    conda install python=3.8 numpy pybind11 cython pytest sphinx matplotlib mypy compilers
+
+    # Build SciPy for development work plus run tests
+    python runtests.py    # Alternatively, it's fine to use `python setup.py develop`
```

Comment:
There should be a `git clone` step before this, no?

Reply:
I think that's best kept separate. There's multiple logically distinct parts that one could have:

1. Installing Python
2. Installing Git & forking/cloning
3. Installing dependencies
4. Building SciPy
5. Running tests

There's multiple ways to do (1) and (2), and most devs will have that setup already or know how to do it in their own preferred way - because it's the same for pretty much any project. Hence quickstart instructions should leave those steps out completely, and full instructions should separate them well to not duplicate things over and over.